### PR TITLE
Update to Cardinal 2.2.5-9

### PR DESCRIPTION
**Pull request by agedd**
### Summary of changes

- Update Cardinal version to 2.2.5-9
- Updated the CHANGELOG.md to CardinalMobile SDK upgrade w/ code signing and privacy manifest requirements
- Tested the 3DS flow end-to-end on the demo app and verified it's looking good.

### Checklist

- [x] Added a changelog entry

- @agedd

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## unreleased
 * Require Xcode 15.0+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
+* Update Cardinal SDK to version 2.2.5-9
```
Comment (jaxdesmarais) [inline CHANGELOG.md:5]: Lets add this under a `BraintreeThreeDSecure` section with a note that it includes code signing and privacy manifest requirements

File: Podfile.lock [hunk 2]
```diff
@@ -37,4 +37,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 224cc2d6666b79d3d6adb45cccd6dfbc6fe74d18
 
-COCOAPODS: 1.14.2
+COCOAPODS: 1.15.2
```
Comment (jaxdesmarais) [inline Podfile.lock:40]: If we want to make this change lets pull it into a separate PR

**Comment by jaxdesmarais** [comment]
I assume we've also gone through the 3DS flow in the demo app to ensure there are no issues? I didn't see it in the PR description so just wanted to be sure.

File: CHANGELOG.md (hunk 1 above)
Comment (agedd) [reply CHANGELOG.md]: for sure, addressed in 3439b8d94359bcc9bed284d094168a8bde1b6084

**Comment by agedd** [comment]
> I assume we've also gone through the 3DS flow in the demo app to ensure there are no issues? I didn't see it in the PR description so just wanted to be sure.

ohh that's a good callout, thanks! i just went through the 3DS flow end-to-end and it's looking good, added this note in the PR description

**Review by jaxdesmarais — APPROVED** [review]
